### DISCUSSION TITLE
Samuel Willyanto | Samu

### DISCUSSION THREAD

**Discussion by samuelwillyanto** [onlyphantom/llm-python#37] [Practicum: Generative AI]
Hi, this is my submission from the last workshop
https://colab.research.google.com/drive/1Nq1NXq9vT414r8EhjxF9F52tlY3RAmXI?usp=sharing#scrollTo=i_SdqPeOZkH1

Thanks

**Comment by onlyphantom** [comment]
Well done Samuel! Marked as passed and will be issuing your certificate soon!
